reinforcement_learning/rl_cartpole_batch_coach/rl_cartpole_batch_coach.ipynb

```diff
@@ -0,0 +1,726 @@
+{
```

Comment:
<blockquote>Assume the updated policy is now deployed, we can use Batch Transform to collect rollout data from this policy.</blockquote><p><strong>Assuming</strong> the updated policy is now deployed, we can use Batch Transform to collect rollout data from this policy.</p><p> </p><p>I think it is worth mentioning that the Batch Transform method works well when you have episodes concurrently interacting with the policy. That way you are not using Batch Transform for only one inference at a time.</p><p> </p><p>Another suggestion: we can add an example on how this will work in the real world. We can take an email campaign or phone notifications as an example. This example can either be at the end here, or at the very top to highlight the importance of this notebook. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/926/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>